### PR TITLE
Show participant detail and profile picture

### PR DESCRIPTION
**Pull request by Thanawas-Sirilertsathit**
- Profile picture and name of participants in edit page and detail page
- Show number of participant in edit page
- minor fix for long detail that will go out the box in detail page
- minor fix for card box in edit page when activity has many details, it will unable to scroll up to the top

### CODE REVIEW

File: activities/views.py [hunk 1]
```diff
@@ -185,6 +185,10 @@ def edit_activity(request: HttpRequest, activity_id: int) -> JsonResponse:
         modified_activity = get_object_or_404(models.Activity, pk=activity_id)
 
         # If user has set the date use, set activity date.
+        if max_people and modified_activity.people > max_people:
+            return JsonResponse({"message": "Number of participants exceed the capacity.",
```
Comment (OmegaOoh) [inline activities/views.py:189]: Please add the test for this

**Review by OmegaOoh — CHANGES_REQUESTED** [review]

**Comment by Thanawas-Sirilertsathit** [comment]
image
@TeerapatTrepopsakulsin at least it should tell something that better than undefined. Edit activity restricted to host problem.

**Review by OmegaOoh — APPROVED** [review]
It does meet the acceptance criteria, but we still need to polish it further in the future.

**Review by TeerapatTrepopsakulsin — APPROVED** [review]
- Profile picture and name of participants in edit page and detail page _Approve_
- Show number of participant in edit page _Approve_
- minor fix for long detail that will go out the box in detail page _Approve_
- minor fix for card box in edit page when activity has many details, it will unable to scroll up to the top _Approve_

**Review by opxz7148 — APPROVED** [review]
Look good participant profile show properly.

**Pull request merged**